filament/backend/include/backend/DriverEnums.h

```diff
@@ -505,6 +505,20 @@ enum class TextureSwizzle {
     CHANNEL_3
 };
 
+//! returns whether this format a depth format
+static constexpr bool isDepthFormat(TextureFormat format) noexcept {
```

Comment:
Note that several backends, including GL, can now use this in their impl of `isTextureFormatMipmappable()`.